README.md

```diff
@@ -13,6 +13,11 @@ var path = require("path");
 const config = {
   // path to your assets
   assetsPath: path.join("app", "assets", "images"),
+  // asset gets copied to this folder if provided
+  assetsOutputPath: path.join("public/copies"),
+  // link to assetsOutputPath with hash goes here
+  // this is only useful when you copy the file using assetsOutputPath
```

Comment:
might be worth providing context as to why this is useful (e.g. reference the cache key)